src/index.ts

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import UsersRoutes from './router/users';
+import { ExpressJoiError } from 'express-joi-validation';
+
+const port = 3000;
+const app = express();
+
+app.use(express.json());
+app.use('/users', UsersRoutes);
+
+app.get('/', (req, res) => {
+    res.send('Server works!');
+});
+
+// Custom error handler
+app.use((err: ExpressJoiError, req: express.Request, res: express.Response) => {
+    if (err && err.error && err.error.isJoi) {
+        const e: ExpressJoiError = err;
```

Comment:
it is not necessary to mention type of e if it is the same as 'error' type.
And also I do not see why we need new variable)